### ISSUE TITLE
Running with composer fails due to requiring autoload.php

### ISSUE THREAD

**Issue by m-idler** [dg/ftp-deployment#143]
Version: 3.3.2, 3.4.0

### Bug Description
When running deployment through composer, execution fails because requiring `autoload.php` fails (`require __DIR__ . '/../vendor/autoload.php';` in deployment.php).

relevant change: https://github.com/dg/ftp-deployment/commit/4bf96600bc8466c46fdfb8dd5dd94a1ef4eed442#diff-1c20baa0d0474ecc14012364394f088a1dbbbb8dee5fa999584a123e841cb4db 

error output:
```
PHP Fatal error:  require(): Failed opening required '/app/vendor/dg/ftp-deployment/src/../vendor/autoload.php' (include_path='.:/usr/local/lib/php') in /app/vendor/dg/ftp-deployment/src/deployment.php on line 12
```

### Steps To Reproduce
Could either be reproduced by calling deployment directly:
`vendor/bin/deployment deployment.php`
or by defining it as a script within composer.json:
```json
{
  ...
  "scripts": {
    ...
    "deploy": "deployment deployment.php"
    ...
  }
  ...
}
```
and run it with composer: `composer deploy`

### Expected Behavior
Up to version 3.3.1 it was working as expected 

### Possible Solution
I guess requiring the autoload.php is necessary when packing it as phar? Maybe there is a possibility to distinguish in which environment the  script is called? When calling it through composer, requiring the autoload file doesn't seem to be necessary at all

**Comment by dg** [comment]
I didn't expect anyone to install it via the "composer require", so I hope the fix will help.

**Comment by dasim** [comment]
> I didn't expect anyone to install it via the "composer require", so I hope the fix will help.

I also use it via `composer require` for a long time and this problem occured recently to me as well. IMO your fix doesn’t help as only this line needs to be left out:

`require __DIR__ . '/../vendor/autoload.php';`

The other `require` lines are still needed.

**Comment by dg** [comment]
It should be working now.

**Comment by kravcik** [comment]
@dg - can you realease some tag with this fix please?  3.4.1 (for PHP >=7.1)

**Comment by dg** [comment]
done
